src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    if (action.type === 'clear') {
```

Comment:
1. use object destructing for `action`
2. it'd much better to use` switch` case here instead of `if`
3. it is a good practice when you add a default behavior for `switch` statement. Yet, if you don't have such one - just throw an Error with text like "unexpected action"